README.md

```diff
@@ -11,4 +20,18 @@ VITE_UPLOAD_PRESET='YOUR-UNSIGNED-UPLOAD-PRESET'
 
 # Run
 
-`npm run dev`
+From your terminal, in the **react-pa** folder, run: 
```

Comment:
if they clone the repository the folder name will be react-photo-album so we should either state that or tell them to use the folder name they chose

Reply:
Addressed in a suggestion further down.